### PR TITLE
Add brainmapper widget docs

### PR DESCRIPTION
**Pull request by adamltyson**
This PR adds:
- Basic user documentation
- A tutorial
- A blog announcing the new plugin
- Some references in other sections

### CODE REVIEW

File: docs/source/documentation/brainglobe-utils/transform-widget.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+```{hint}
+If you're not sure, we recommend installing the metapackage
+```
+
+## How to use
+
+### Prerequisites 
+Before you use this widget you need three things:
+1. Your raw data
```
Comment (alessandrofelder) [inline docs/source/documentation/brainglobe-utils/transform-widget.md:44]:
```suggestion
1. Your raw image data
```
More specific? (I was a bit confused about the term when I was reviewing the widget PR)

File: docs/source/documentation/brainglobe-utils/transform-widget.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+2. The [brainreg](/documentation/brainreg/index) output directory, generated by registering your raw data to a BrainGlobe atlas
+3. Some points defined in the raw data space, e.g. cells detected by [cellfinder](/documentation/cellfinder/index). 
+
+```{hint}
+This widget will support any points that can be loaded into napari. Many file formats (e.g. the BrainGlobe xml format) 
+can be loaded by dragging and dropping them onto the napari window.
```
Comment (alessandrofelder) [inline docs/source/documentation/brainglobe-utils/transform-widget.md:50]: Nice!

File: docs/source/documentation/brainglobe-utils/transform-widget.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+5. Click `Transform points` and when prompted, select your brainreg output directory
+
+## Results
+
+### Transformed points
+This widget will produce two new layers:
+- `Points in downsampled space` - this is the points layer, but each point is transformed to the same space as the 
+`downsampled.tiff` image generated by brainreg. This is the same coordinate space as the raw data, but resampled and 
+realigned to match the atlas. This is saved for completeness, but likely is not needed for most analyses. 
+- `Points in downsampled space` - this is the points layer, but each point is transformed to the atlas space.
```
Comment (alessandrofelder) [inline docs/source/documentation/brainglobe-utils/transform-widget.md:67]:
```suggestion
- `Points in atlas space` - this is the points layer, but each point is transformed to the atlas space. 
```
Copy-paste typo :)

**Review by alessandrofelder — APPROVED** [review]
Looking good - tiny comments.

File: docs/source/blog/brainmapper-widget.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+it easier to iteratively optimise parameters. However, to assign cells to a brain region, users had to use the 
+`brainmapper` command line tool. 
+
+To fix this **we have released a new [`brainmapper` napari widget](/documentation/brainglobe-utils/transform-widget)**. 
+This allows users to combine the results of 
+`brainreg` and `cellfinder` to analyse the distribution of cells across the brain.
+
+## What does it do?
+
+The plugin takes as input cell coordinates (from `cellfinder` or another tool) and a `brainreg` output directory. It
```
[the remark this answers is not in the document]
Comment (adamltyson) [reply docs/source/blog/brainmapper-widget.md:32]: They should have the raw data to have used brainreg in the first place. I think in the blog it's ok to leave it at a high level, the docs outline the specific requirements.

**Comment by adamltyson** [comment]
Thanks @alessandrofelder, good catches!

**Pull request merged**